<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegressionDiag</class>
 <widget class="QDialog" name="RegressionDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Slope:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_slope">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="value" stdset="0">
        <double>99999999.000000000000000</double>
       </property>
       <property name="intValue" stdset="0">
        <number>99999999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcdNumber_offset">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="value" stdset="0">
        <double>99999999.000000000000000</double>
       </property>
       <property name="intValue" stdset="0">
        <number>99999999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Slope standard error:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNumber_StdError">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="value" stdset="0">
        <double>99999999.000000000000000</double>
       </property>
       <property name="intValue" stdset="0">
        <number>99999999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Offset:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Count:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumbe_count">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="intValue" stdset="0">
        <number>99999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RegressionDiag</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RegressionDiag</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
